.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,3 +39,13 @@ In addition to the feature you are implementing, have you checked the following:
 #### Should this PR be mentioned in Design System updates?
 
 #### Is this change backwards compatible or is it a breaking change?
+
+<!---
+#### Notifications
+
+[Update/New tag]Exact name of component/template
+
+[name of the updated section within the guidance][Another update section]
+
+[Message to fill the inline notification]
+-->
```

Comment:
This is helpful.

Can we pull this up to the top of the template?

Also, can you provide a filled in example that accompanies this template?

Reply:
Yes, done in latest commit